src/common/src/array/stream_chunk.rs

```diff
@@ -70,9 +72,8 @@ pub type Ops<'a> = &'a [Op];
 #[derive(Clone, PartialEq)]
 pub struct StreamChunk {
     // TODO: Optimize using bitmap
-    ops: Vec<Op>,
-
-    pub(super) data: DataChunk,
+    ops: Arc<[Op]>,
```

Comment:
Can we use `Box<[Op]>` here?

Reply:
Just wanted to make StreamChunk cheaply cloneable.